### PR TITLE
tweak Java 1.8 wrapper to use different Java version on POWER systems

### PR DESCRIPTION
**Pull request by edmondac**

### CODE REVIEW

**Comment by edmondac** [comment]
I realise that we've got Java 11 from OpenJDK these days, but this allows Java 8 from OpenJDK. It uses the new framework feature for choosing a version based on the architecture of the system - so in this case Intel vs. POWER

File: easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'Java'
+easyblock = 'ConfigureMake'
+version = '1.8_191-b26-OpenJDK'
+
+homepage = "https://openjdk.java.net/"
+description = """An open-source implementation of the Java Platform, Standard Edition"""
+
+toolchain = {'name': 'GCCcore', 'version': 'system'}
```
Comment (boegel) [inline easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb]: @edmondac This should be `toolchain = SYSTEM`?

Cfr. https://easybuild.readthedocs.io/en/latest/System_toolchain.html

File: easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/corba/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jaxp/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jaxws/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/langtools/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jdk/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/nashorn/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/archive/',
+]
+sources = [
```
Comment (boegel) [inline easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb]: @edmondac Maybe include a comment how these were determined, and how they relate to the version you set?

File: easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb (hunk 1 above)
Comment (edmondac) [reply easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb]: Ahah!

File: easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb (hunk 2 above)
Comment (edmondac) [reply easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb]: Done

**Comment by boegel** [comment]
@edmondac So, is this still WIP?

**Comment by edmondac** [comment]
> @edmondac So, is this still WIP?

Ah - no! Finally done :-)

**Comment by boegel** [comment]
@edmondac Can you look into submitting a test report that shows that this works as expected on POWER?

Preferably in a context where neither `Java/1.8` nor the underlying `Java` are available yet, so the test report shows the different underlying Java being used on POWER...

(mostly for future reference, it's not that I want proof that it works)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3146.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/efbc3ca649cab399a91aa6ac30a3842d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2630.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/316b2760ba1156048ed7fa32f2a26539 for a full test report.

**Comment by edmondac** [comment]
Test report by @bear-software
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/46aa587a31c12a4273f54332b591eae4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @edmondac!

**Pull request merged**